network/nodenetwork/nodekeeper.go

```diff
@@ -303,6 +304,7 @@ func (nk *nodekeeper) Sync(ctx context.Context, nodes []insolar.NetworkNode, cla
 	}
 
 	if nk.shouldExit(foundOrigin) {
+		nk.Abort()
```

Comment:
Remove this call, `Abort` method and termination handler injection. Termination is handled on higher layers and error return is enough.

Reply:
can't remove injection - we need to know, when leave claim approved